audio_service/lib/audio_service.dart

```diff
@@ -1251,16 +1281,16 @@ class AudioService {
 /// This class is deprecated. Use [BaseAudioHandler] instead.
 @deprecated
 abstract class BackgroundAudioTask extends BaseAudioHandler {
-  /// Deprecated
+  /// Deprecated. Use [AudioServiceConfig.fastForwardInterval] from [AudioService.config] instead.
   Duration get fastForwardInterval => AudioService.config.fastForwardInterval;
 
-  /// Deprecated
+  /// Deprecated. Use [AudioServiceConfig.rewindInterval] from [AudioService.config] instead.
   Duration get rewindInterval => AudioService.config.rewindInterval;
 
   /// Depricated. Replaced by [AudioHandler.stop].
   @mustCallSuper
-  Future<void> onStop() async {
-    await super.stop();
+  Future<void> onStop() {
```

Comment:
My preference for block bodies is to always use `async`, and for functional bodies to return the future "unless" arguments need to be awaited. So in this case, I would be happy with either of these:

```dart
Future<void> onStop() async {
  await super.stop();
}
```

or

```dart
Future<void> onStop() => super.stop();
```